src_new/util.js

```diff
@@ -328,18 +328,25 @@ exports.resizeWindow = function(theDocument, width, height, divId){
 	/* istanbul ignore else */
 	if(height && width){
 		try{
-			var fr = theDocument.defaultView.frameElement;
+			var defaultViewFrame = theDocument.defaultView.frameElement;
 			if(divId){
-				 var element = document.getElementById(divId);
-				 var ele = element.querySelector("div");
-				 ele.style.height = ""+ height + "px";
-				 ele.style.width = ""+ width+ "px";
-				 fr = element.querySelector("iframe");
+				var adSlot = document.getElementById(divId);
+				var adSlot_Div = adSlot.querySelector("div");
+				var adSlot_Div_iframe = adSlot_Div.querySelector("iframe");
+				[adSlot_Div,adSlot_Div_iframe].forEach(function(ele){
+					if(ele){
+						ele.width ="" +  width;
+						ele.height ="" + height;
+						ele.style.width = "" + width + "px";
+						ele.style.height = "" + height + "px";
+					}
+				});
+				defaultViewFrame = adSlot.querySelector("iframe");
 			}
-			fr.width ="" +  width;
-			fr.height ="" + height;
-			fr.style.width = "" + width + "px";
-			fr.style.height = "" + height + "px";
+			defaultViewFrame.width ="" +  width;
```

Comment:
execute forEach on array here with defaultViewFrame as an element in array, this will avoid repeating code for changing style of elements.

Reply:
I've resused the code for resizeWindow and its specific to when we pass divId
rest functionality is untouched

Reply:
I am trying to tell you that on line 338 to 341 you are changing style of elements in an array.
You can declare this array at line 331, then push elements(adSlot, adSlot_Div, etc) to this array and at line 346 you can change style of all elements in array.